### PR TITLE
add cjis to compliance demo

### PR DESCRIPTION
**Pull request by l3acon**
This should be fairly simple, though I had some issues during testing. On RHEL7 I encountered jobs that failed without any error messages. I would normally chalk it up to the random gremlins but it happened more than once. Maybe @jce-redhat or someone has seen something similar?

Anyway, this at least worked once on rhel7 and rhel8.

### CODE REVIEW

File: linux/cjis-prerequisites.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+# The CJIS role seems to assume these packages are installed and the
```
Comment (jce-redhat) [inline linux/cjis-prerequisites.yml:2]: @l3acon i haven't tested this yet, but i took a look at the CJIS role and i believe you won't need to install and enable firewalld and NetworkManager if the following variables get set:

```
service_firewalld_enabled: false
firewalld_sshd_port_enabled: false
```

these can be added to the [LINUX / Multi-profile Compliance](https://github.com/ansible/product-demos/blob/10f0bb46414c56481a8fb0e58a4d927f2c9b2235/linux/setup.yml#L362) job template as long as the other CaC roles don't use variables of the same name.

File: linux/compliance-enforce.yml [hunk 2]
```diff
@@ -10,6 +10,11 @@
       ansible.builtin.assert:
         that: "ansible_os_family == 'RedHat'"
 
+    - name: Include prerequisites for cjis profile
+      ansible.builtin.include_tasks:
+        file: cjis-prerequisites.yml
+      when: compliance_profile == 'cjis'
+
```
Comment (jce-redhat) [inline linux/compliance-enforce.yml:17]: if you add "cjis" to the [survey spec](https://github.com/ansible/product-demos/blob/10f0bb46414c56481a8fb0e58a4d927f2c9b2235/linux/setup.yml#L384) for the "LINUX / Multi-profile compliance" job template definition, you shouldn't need to add this separate task for calling the CJIS role.

**Review by jce-redhat — CHANGES_REQUESTED** [review]
added a couple of ideas for you to try, lemme know if you want help testing

**Review by jce-redhat — APPROVED** [review]

**Comment by l3acon** [comment]
Yay thanks @jce-redhat

**Pull request merged**